src/ace/ari_text/parsemod.py

```diff
@@ -85,16 +85,23 @@ def p_typedlit_am(p):
 
 
 def p_typedlit_tbl_empty(p):
-    '''typedlit : SLASH TBL structlist'''
+    '''typedlit : SLASH TBL structpair'''
     ncol = int(p[3].get('c', 0))
     table = Table((0, ncol))
     p[0] = LiteralARI(type_id=StructType.TBL, value=table)
 
 
 def p_typedlit_tbl_rows(p):
-    '''typedlit : SLASH TBL structlist rowlist'''
-    ncol = int(p[3].get('c', 0))
+    '''typedlit : SLASH TBL structpair rowlist'''
+    ncol = 0
     nrow = len(p[4])
+    col_count = p[3].get('c')
+    if isinstance(col_count, LiteralARI):
+        ncol = int(str(col_count.value))
+    elif isinstance(col_count, str):
+        ncol = int(col_count)
+    else:
+        raise ParseError(f"Unexpected type for column count: {type(col_count)}")
```

Comment:
This form gets duplicated a few times, and probably should be refactored up into the `structpair` defintion or even a refactor of `p_ssp_primitive` to enable an untyped literal symbol. Let me know if you want help with this.